entity-service-client/src/main/java/org/hypertrace/entity/service/client/config/EntityServiceClientCacheConfig.java

```diff
@@ -10,28 +10,39 @@
  */
 public class EntityServiceClientCacheConfig {
 
+  public static long DEFAULT_CACHE_REFRESH_MS = 240000L;
```

Comment:
Will this enable expiry to extend by another 5 mins? Does this mean that we don't want to expire the entry?

Reply:
So, if the replacement has happened, it will continue to stay in memory, But, if a replacement has not happened, it will be expired, right?

Reply:
> So, if the replacement has happened, it will continue to stay in memory, But, if a replacement has not happened, it will be expired, right?
Yes but a small correction. expiry in guava cache doesn't mean the entry will be cleaned up from the cache automatically. There is no auto cleanup of expired entries. Its just that once expired, the stale entry in the cache is not returned and rather the loader is invoked to get the fresh value. Entries are evicted from the cache only based on the size.

Reply:
Got it.